satellite/satellitedb/migrate.go

```diff
@@ -1396,6 +1396,39 @@ func (db *DB) PostgresMigration() *migrate.Migration {
 					`ALTER TABLE graceful_exit_transfer_queue ADD COLUMN order_limit_send_count integer NOT NULL DEFAULT 0;`,
 				},
 			},
+			{
+				DB:          db.db,
+				Description: "Add stripecoinpayments_tx_conversion_rates",
+				Version:     68,
+				Action: migrate.SQL{
+					`CREATE TABLE stripecoinpayments_tx_conversion_rates (
+						tx_id text NOT NULL,
+						rate bytea NOT NULL,
+						created_at timestamp with time zone NOT NULL,
+						PRIMARY KEY ( tx_id )
+					);`,
+				},
+			},
+			{
+				DB:          db.db,
+				Description: "Add timeout field to coinpayments_transaction",
+				Version:     69,
+				Action: migrate.SQL{
+					`DROP TABLE coinpayments_transactions CASCADE`,
```

Comment:
I understand that at this point this is OK, there is no data to conserve, right?

Reply:
yup